worlds/AutoWorld.py

```diff
@@ -44,6 +45,12 @@ def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> Aut
                     dct["required_client_version"] = max(dct["required_client_version"],
                                                          base.__dict__["required_client_version"])
 
+        # create missing options_dataclass from legacy option_definitions
+        # TODO - remove this once all worlds use options dataclasses
+        if "options_dataclass" not in dct and "option_definitions" in dct:
```

Comment:
Seems to me this check currently fails since `World` defaults `options_dataclass = PerGameCommonOptions` so all the worlds will have it in their dct?
![Screenshot_32](https://user-images.githubusercontent.com/13184667/218884631-c19f5d99-fca0-4118-be9c-415ae39a5942.png)

Reply:
nevermind that was `AutoWorld` itself lmao. it's in `bases` so yeah this works